CONTRIBUTING.md

```diff
@@ -33,13 +33,40 @@ yarn build
 yarn build:watch
 ```
 
-## Run tests
+## Updating and adding packages
+
+Since this is a monorepo with several subrepos, if you want to update/add packages in the subrepos like `create-snowpack-app/app-template-11ty` you'll want to run the commands in the target subdirectory like
```

Comment:
I never head the temr "subrepos". I think I'd just call it "packages"

Reply:
probably because I made it up because I couldn't remember "packages" lol